backend/src/packages/projects/project.controller.ts

```diff
@@ -35,6 +35,15 @@ import { type ProjectUploadImageParametersDto } from './libs/types/types.js';
  *            type: number
  *            format: int64
  *            minimum: 1
+ *          projectType:
+ *          type: string
+ *          enum:
+ *          - eCommercial
```

Comment:
```suggestion
 *          - e-commercial
```
should be in kebab case